src/panels/lovelace/hui-root.ts

```diff
@@ -412,6 +412,8 @@ class HUIRoot extends LitElement {
         newSelectView = 0;
       } else if (viewPath === "hass-unused-entities") {
         newSelectView = "hass-unused-entities";
+      } else if (viewPath === "default_view") {
+        newSelectView = undefined;
```

Comment:
Why would we do this? Wouldn't this prevent navigating back to the view with path `default_view`?

Reply:
The statement underneath this would already cover this right? And it is no certainty that the `default_view` is also the first view.

Reply:
Yeah, I think you're right. I think checking if viewIndex was set to 0 or undefined at line 621 was more clear in its intention as I first proposed. What do you think?

Reply:
I would add your new logic at line 411, that's where we are choosing a view when no view is specified.